rpm/checker.go

```diff
@@ -0,0 +1,77 @@
+package rpm
+
+import (
+	"fmt"
+
+	"github.com/facebookincubator/nvdtools/wfn"
+)
+
+// Checker knows how to verify whether some package has been fixed or not
+type Checker interface {
+	// Check should return whether a given package on distribution is fixed for some CVE
+	Check(pkg *Package, distro *wfn.Attributes, cve string) bool
+}
+
+// CheckAny returns a Checker which will return true if any of the underlying checkers returns true
+func CheckAny(chks ...Checker) Checker {
+	return anyChecker(chks)
+}
+
+type anyChecker []Checker
+
+// Check is part of the Checker interface
+func (c anyChecker) Check(pkg *Package, distro *wfn.Attributes, cve string) bool {
+	for _, chk := range c {
+		if chk.Check(pkg, distro, cve) {
+			return true
+		}
+	}
+	return false
+}
+
+// CheckAll returns a Checker which will return true if all of the underlying checkers returns true
+func CheckAll(chks ...Checker) Checker {
+	return allChecker(chks)
+}
+
+type allChecker []Checker
+
+// Check is part of the Checker interface
+func (c allChecker) Check(pkg *Package, distro *wfn.Attributes, cve string) bool {
+	if len(c) == 0 {
+		return false
```

Comment:
why false? seems logical to assume that all of none checkers returned true and return true. would also make this check redundant...